tests_e2e/orchestrator/lib/agent_test_suite.py

```diff
@@ -450,7 +450,17 @@ def _setup_test_nodes(self, test_context: AgentTestContext) -> None:
             #
             log.info("Extracting %s on the test node", self._test_tools_tarball_path)
             command = f"tar xvf {target_path/self._test_tools_tarball_path.name}"
-            log.info("Remote command [%s] completed:\n%s", command, ssh_client.run_command(command))
+            try:
+                log.info("Remote command [%s] completed:\n%s", command, ssh_client.run_command(command))
+            except CommandError as e:
+                # Some distros do not have tar installed by default. Install it if needed.
+                if "tar: command not found" in e.stderr:
+                    log.info("Failed to extract waagent-tools.tar on the test node because tar is not installed. Installing tar...")
+                    ssh_client.run_command("dnf -y install tar", use_sudo=True)     # Alma 8 is the only image without tar by default, so using dnf. If needed, we can add logic to use other package managers as needed.
```

Comment:
get_distro.py is in the tar, so I didn't limit this change to alma_8. If we need distro specific logic, we can add that as needed